pkg/gofr/cmd.go

```diff
@@ -62,9 +77,18 @@ func (cmd *cmd) handler(path string) Handler {
 	return nil
 }
 
-func (cmd *cmd) addRoute(pattern string, handler Handler) {
-	cmd.routes = append(cmd.routes, route{
-		pattern: pattern,
-		handler: handler,
-	})
+func (cmd *cmd) addRoute(r route) {
+	cmd.routes = append(cmd.routes, r)
+}
+
+func (cmd *cmd) printHelp() {
+	fmt.Println("Available commands:")
+	for _, route := range cmd.routes {
+		help := route.help
+		if help == "" {
+			help = route.description // Use description if custom helper documentation is not provided
+		}
+		fmt.Printf("  %s: %s\n", route.pattern, help)
```

Comment:
Here we can try to print the description and help doc together in 2 separate lines. It would help if you could paste some screenshots of how the output would look like.